packages/contracts-bedrock/src/universal/OptimismMintableERC20Factory.sol

```diff
@@ -32,18 +34,25 @@ contract OptimismMintableERC20Factory is ISemver {
     ///         the OptimismMintableERC20 token contract since this contract
     ///         is responsible for deploying OptimismMintableERC20 contracts.
     /// @notice Semantic version.
-    /// @custom:semver 1.8.0
-    string public constant version = "1.8.0";
+    /// @custom:semver 1.9.0
+    string public constant version = "1.9.0";
 
+    /// @notice Constructs the L1StandardBridge contract.
+    constructor() {
+        initialize({ _bridge: address(0) });
+    }
+
+    /// @notice Initializes the contract.
```

Comment:
There is some boiler plate we've used in the past to discourage using the immutable cased getter. We should add that here.

ie. https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/L1/L2OutputOracle.sol#L18-L19

Reply:
great call. thank you!